lib/mock-component/mock-component.spec.ts

```diff
@@ -194,6 +194,15 @@ describe('MockComponent', () => {
       customFormControl.__simulateChange('foo');
       expect(component.formControl.value).toBe('foo');
     });
+
+    it('should allow to get control value', () => {
+      fixture.detectChanges();
+      component.formControl.setValue('foo')
+      const customFormControl: MockedComponent<CustomFormControlComponent> = fixture.debugElement.query(
+        By.css('custom-form-control')
+      ).componentInstance;
+      expect(customFormControl.__value).toBe('foo');
+    });
```

Comment:
please check my comments in the PR, perhaps it makes sense to update the readme with an explanation how to fetch a value from a mocked form control.